common/persistence/nosql/nosqlplugin/interfaces.go

```diff
@@ -49,13 +49,13 @@ type (
 	// Cassandra implementation cannot do it due to backward-compatibility. Any other NoSQL implementation should use datablob for non-significant columns.
 	// Follow the comment for each tableCRUD for what are 'significant' columns.
 	tableCRUD interface {
-		historyEventsCRUD
```

Comment:
Could we add a mockgen for this interface file?

Reply:
We can add it when we are writing unit tests, it's pretty easy to have. 
But I don't think we should do it now as we don't have any unit tests here.
Because having mockgen would just become an unnecessary burden to maintain the code.

Reply:
I will try to add some unit tests later